test/tests/functional/pbs_user_reliability.py

```diff
@@ -41,6 +41,11 @@
 
 class Test_user_reliability(TestFunctional):
 
+    def setUp(self):
+        TestFunctional.setUp(self)
+        a = {'resources_available.ncpus': 4}
+        self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
```

Comment:
Now that you are setting the number of cpus, you need to skip these tests on cpuset machines.